eventsourcing/infrastructure/cassandra/activerecords.py

```diff
@@ -69,14 +69,22 @@ def get_items(self, sequence_id, gt=None, gte=None, lt=None, lte=None, limit=Non
         return items
 
     def all_items(self):
+        for record in self.all_records():
+            yield self.from_active_record(record)
+
+    def all_records(self):
         query = self.active_record_class.objects.all().limit(10)
         page = list(query)
         while page:
             for record in page:
-                yield self.from_active_record(record)
+                yield record
             last = page[-1]
             page = list(query.filter(pk__token__gt=Token(last.pk)))
 
+    def delete_record(self, record):
```

Comment:
Well, it doesn't make something possible that was previously impossible. :-) I added this method to check the snapshots are working in all DBMSs. There is room for improvement in the snapshot tests, and maybe this could be removed. Or maybe it should be accompanied by a method to update an event? I'm not sure...